chatterbot/storage/mongodb.py

```diff
@@ -124,6 +124,7 @@ def filter(self, **kwargs):
             or_regex = '|'.join([
                 '{}'.format(word) for word in search_text_contains.split(' ')
             ])
+            or_regex = re.escape(or_regex)
```

Comment:
It looks like this change is resulting in a test failure:
```
======================================================================
FAIL: test_search_text_contains_multiple_matches (tests.storage.test_mongo_adapter.MongoAdapterFilterTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/gunthercox/ChatterBot/tests/storage/test_mongo_adapter.py", line 320, in test_search_text_contains_multiple_matches
    self.assertEqual(len(results), 2)
AssertionError: 0 != 2
```

This would need to be corrected before I could merge this.

Reply:
@gunthercox Fixed it, to support regexes with | in them, the expression must be parenthesized. By the way I need your review at: [this pull request](https://github.com/gunthercox/ChatterBot/pull/1732)

Reply:
@gunthercox Dismiss this pull request untill i fix it, my last one didn't work in practice

Reply:
@gunthercox I tried a few commits but didn't work, I don't know how to solve this to be honest, here's something might help: [stackoverflow question](https://stackoverflow.com/questions/280435/escaping-regex-string-in-python)

Reply:
@minjae1025 Maybe you know how to fix it?

Reply:
@gunthercox Fixed, by the way I need your review at: [this pull request](https://github.com/gunthercox/ChatterBot/pull/1732)